### PR TITLE
Deprecate Grafana Phlare

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

**Review by cyriltovena — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,69 +1,7 @@
-# <p align="center">Grafana Phlare</p>
+# Grafana Phlare is archived
 
-<p align="center"><img src="images/logo.png" alt="Grafana Phlare logo" width="400"></p>
+On 2023-03-15, [Grafana Labs acquired Pyroscope](https://blog/2023/03/15/pyroscope-grafana-phlare-join-for-oss-continuous-profiling/).
```
Comment (na--) [inline README.md:3]:
```suggestion
On 2023-03-15, [Grafana Labs acquired Pyroscope](https://grafana.com/blog/2023/03/15/pyroscope-grafana-phlare-join-for-oss-continuous-profiling/).
```

**Review by simonswine — APPROVED** [review]
LGTM

**Review by na-- — CHANGES_REQUESTED** [review]
See https://github.com/grafana/phlare/pull/588/files#r1136947431, the blog link is currently broken

**Pull request merged**